WBooks/TableBooks/TableBookController.swift

```diff
@@ -10,37 +10,46 @@ import UIKit
 
 final class TableBookController: UITableViewController {
     private lazy var tableBookView = TableBookView()
+    private let bookViewModel: TableBookViewModel
```

Comment:
Looks great, but let's rename this property so that it matches with the name of the class:
```Swift
private let tableBookViewModel: TableBookViewModel
```